### DISCUSSION TITLE
Idea: Sorting of electricity consumption and carbon emissions.

### DISCUSSION THREAD

**Discussion by MichaelWMcLeod** [electricitymaps/electricitymaps-contrib#5380] [Ideas]
It would be helpful to be able to sort electricity consumption and carbon emissions, eg. sort them by amount of consumption and emissions.

**Comment by VIKTORVAV99** [comment]
Hi @MichaelWMcLeod and thanks for submitting your idea!

Do you mean that you want the ability to sort zones in the zones in the list to the left by consumption (load) and their total emissions?

If that is the case I'm not sure it would be a fair comparison as some countries have way more people than other countries and therefore release more carbon in total. But I'm not against discussing it so I'll turn this issue into a discussion as it is better suited for that.

**Comment by Azerty383838** [comment]
Hello, 
It might be interesting to know the total emissions per capita in addition to the total emissions.
